cluster/ci/config/prow/plugins.yaml

```diff
@@ -1094,3 +1097,25 @@ external_plugins:
     - name: needs-rebase
       events:
       - pull_request
+
+bugzilla:
+  default:
+    openshift-4.0:
```

Comment:
These are dead, let's just remove these.  There will never be 4.0 code, I deleted most of the CI jobs, it's gone, etc.

Reply:
done